source/locale/zh_CN/LC_MESSAGES/reference/generated/optuna.storages.RDBStorage.po

```diff
@@ -986,70 +991,70 @@ msgstr ""
 #: optuna.storages._rdb.storage.RDBStorage.set_trial_user_attr:9
 #: optuna.storages._rdb.storage.RDBStorage.set_trial_values:8
 msgid "If no trial with the matching ``trial_id`` exists."
-msgstr ""
+msgstr "如果没有匹配 ``trial_id`` 的 trial 存在的话."
 
 #: of optuna.storages._rdb.storage.RDBStorage.get_study_system_attrs:5
 msgid "Dictionary with the optuna-internal attributes of the study."
-msgstr ""
+msgstr "study 中的 optuna 内部属性的字典."
 
 #: of optuna.storages._rdb.storage.RDBStorage.get_study_user_attrs:5
 msgid "Dictionary with the user attributes of the study."
-msgstr ""
+msgstr "study 中的用户属性词典."
 
 #: of optuna.storages._rdb.storage.RDBStorage.get_trial:5
 msgid "Trial with a matching trial ID."
-msgstr ""
+msgstr "带有匹配的 trial ID 的 trial."
 
 #: of
 #: optuna.storages._rdb.storage.RDBStorage.get_trial_id_from_study_id_trial_number:4
 #: optuna.storages._rdb.storage.RDBStorage.get_trial_number_from_id:9
 msgid "Number of the trial."
-msgstr ""
+msgstr "trial 数目."
 
 #: of
 #: optuna.storages._rdb.storage.RDBStorage.get_trial_id_from_study_id_trial_number:8
 msgid "If no trial with the matching ``study_id`` and ``trial_number`` exists."
-msgstr ""
+msgstr "如果没有匹配 ``study_id`` 和 ``trial_number`` 的 trial 存在."
 
 #: of optuna.storages._rdb.storage.RDBStorage.get_trial_number_from_id:5
 msgid "The trial number is only unique within a study, and is sequential."
-msgstr ""
+msgstr "trial 数只在 study 内是独特的, 而且是递增的."
```

Comment:
I see, but according to "consecutive and unique" I guess it could be "线性递增的", as that's the only way to be "consecutive and unique", "线性" itself does not convey the incremental nature of trial numbers.

Reply:
I see, "线性递增的" sounds good to me